pkg/api/tag_test.go

```diff
@@ -0,0 +1,309 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
```

Comment:
this file wasn't just moved, but also iterated on.

if you want to see what was changed here, please inspect a diff between this file and `debugapi/tag_test.go` (which is removed through this PR)

Reply:
it would have been nice if we would have used `git mv`. i'm not sure why it was not shown as moved in the diff. it can still be done though